application/web/web_main.py

```
@@ -1161,6 +1161,25 @@ def callback():
+            logger.error(
+                "OIDC callback returned no 'sub' claim; skipping user persistence"
+            )
+        else:
+            try:
+                user = db.Node_collection().upsert_user(
+                    google_sub=google_sub,
+                    email=id_info.get("email") or "",
+                    display_name=id_info.get("name"),
+                )
+                session["user_id"] = user.id
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)